# [RiskFactors2AD](https://whve.gitbook.io/riskad)

# The AlzRisk Research Resource

Alzheimer's disease (AD) has become the most common of the progressive neurodegenerative diseases with the steadily increasing prevalence and great difficulties in the pathogenesis analysis of AD presently. Although some encouraging progress have been made in understanding the genetic architecture underlying AD, developing phenotypic traits that predispose or protect individuals to disease remain inconclusive and still require further research. The availability of large amounts of genome wide association data across varied traits affords the opportunity to investigate the relationship between myriad traits and AD. Mendelian randomization (MR) is a method that uses genetic variants as proxies for modifiable risk factors to estimate the correlation from causation with known genetic determinants of disease conditions, which could overcome unmeasurable confusion and reverse causality in observational studies. By combining single nucleotide polymorphisms (SNPs) of AD with exposure factors and adopting Two-sample MR randomization methods, we established an online database that can be used to assess evidence of causality where observational associations exist.

We explore causal relationships between 295 traits across multiple phenotypes versus the International Genomics of Alzheimer's Project (IGAP). For each phenotype, the association with AD was assessed using the inverse variance weighted (IVW), MR Egger, Weighted Median, and Weighted Mode methods. Here, we have applied the following inclusion criteria; genome-wide associated SNPs with a minimum p-value less than 5E-8; SNPs or their proxies (minimum R^2 value = 0.8) present in both the exposure and outcome (AD) datasets; R^2 clumping threshold = 0.001. 

**[Database]Full documentation available here:** [https://whve.gitbook.io/riskad](https://whve.gitbook.io/riskad)

**[Database website backed up on github]**(https://github.com/whve/RFforAD/blob/master/SUMMARY.md) 




